cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -22,7 +22,8 @@ This section contains information on all functions in the Cypher query language.
 
 Related information may be found in <<query-operators>>.
 
-Most functions in Cypher will return `null` if an input parameter is `null`.
+[NOTE]
```

Comment:
Please don't stack multiple admonitions ([NOTE], [CAUTION], etc.) on top of each other.
Either join the two notes in one admonition, or just use a regular paragraph.

Reply:
@jjaderberg I made a second one because these notes have nothing to do with one another. as a reader it feels strange to have two different messages presented in one "look here this is important" box.

...and a regular paragraph doesn't draw enough attention for such an important point.

is there an option C that I can choose?